Sources/Sentry/SentryClient.m

```diff
@@ -546,12 +545,6 @@ - (SentryEvent *_Nullable)prepareEvent:(SentryEvent *)event
         event.dist = dist;
     }
 
-    NSString *environment = self.options.environment;
```

Comment:
I don't know why this bit code was here, to first set the event's environment, then do some other work, then set the event's environment again some lines later? It seems completely unnecessary at least, with the current changes where `self.options.environment` now always has a value.